packages/alert/src/index.tsx

```diff
@@ -85,28 +102,42 @@ const Alert: React.FC<AlertProps> = ({
       hasTrim={defaults.hasTrim}
       className={className}
     >
-      {defaults.hasIcon && (
-        <Icon>{getAlertIconForIntent(defaults.intent)({theme})}</Icon>
-      )}
-      <div>
-        {typeof title === 'string' ? (
-          <Title as="h4" color="intense">
-            {title}
-          </Title>
-        ) : (
-          title
-        )}
-        {typeof children === 'string' ? (
-          <Description>{children}</Description>
-        ) : (
-          children
+      <BoxInner
+        appearance={defaults.appearance}
+        hasIcon={defaults.hasIcon}
+        closeOnClick={closeOnClick}
+      >
+        {defaults.hasIcon && (
+          <Icon>{getAlertIconForIntent(defaults.intent)({theme})}</Icon>
         )}
-      </div>
+        <div>
+          {typeof title === 'string' ? (
+            <Title as="h4" color="intense">
+              {title}
+            </Title>
+          ) : (
+            title
+          )}
+          {typeof children === 'string' ? (
+            <Description>{children}</Description>
+          ) : (
+            children
+          )}
+        </div>
+      </BoxInner>
+      <CloseIcon closeOnClick={closeOnClick} />
     </Box>
   )
 }
 
-export const StyledAlert = {Box, Title, Icon, Description}
+export const StyledAlert = {
```

Comment:
Let's change this to `AlertElements` and keep that naming in other components. Here is related issue https://github.com/eyedea-io/smashing-ui/issues/77